ooyala_player/ooyala_player.py

```diff
@@ -351,7 +351,7 @@ class OoyalaPlayerBlock(OoyalaPlayerMixin, XBlock):
         display_name=_("Content Id"),
         help=_("Identifier for the Content Id."),
         scope=Scope.content,
-        default='6068614205001'
+        default='6110589153001'
     )
```

Comment:
If it could be updated/replaced again in the future, then I think we should get it from some setting file.

Reply:
Now it will stay permanent as we've completely moved to brightcove